getnacelle/nacelle-js
# feat: error handling

## Why are these changes introduced?

Addresses [ENG-8502](https://nacelle.atlassian.net/browse/ENG-8502) 



## What is this pull request doing?



Adds error handling to the commerce queries plugin
Modifies the shape of the returned object to be the direct data instead of `{data, error}`
## How to Test


- `git checkout ENG-8502/error-handling-plugin`
- Run `npm run coverage` all tests should pass with 100% coverage



## Checklist



[ENG-8502]: https://nacelle.atlassian.net/browse/ENG-8502?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
